File: web/src/app/Models/Dtos/Job/JobCreate-dto.ts
import { EmploymentType } from "../../Enums/EmploymentType"

export class JobCreateDto{
    companyId: string = ""
    title: string = ""
    description: string = ""
    shortDescription = ""
    EmploymentType: EmploymentType | null = null
    location: string = ""
    tags: string[] = []
    //companyName: string = ""
}

File: backend/SkillProof/SkillProof.Entities/Dtos/Jobs/JobCreateDto.cs
using System.ComponentModel.DataAnnotations;
using SkillProof.Entities.Enums;

namespace SkillProof.Entities.Dtos.Job;

public class JobCreateDto
{
    
    public string Title { get; set; }
    public string Location { get; set; }
    public EmploymentType EmploymentType { get; set; }
    public string? Salary { get; set; }
    public string Description { get; set; }
    public string Tags { get; set; }
}